package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "caver-js",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
do you want to change this too?

Reply:
I did not manually modify this.
Already in package.json, `version` is `v1.9.1`, and when i installed new library, updated version is applied to package-lock.json file.

Reply:
okay. good to go!